go.mod

```diff
@@ -7,6 +7,7 @@ module github.com/hyperledger/aries-framework-go
 require (
 	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
+	github.com/golang/mock v1.3.1
```

Comment:
do we need golang mock

Reply:
./pkg/didcomm/common/service/mocks/mocks.go:	gomock "github.com/golang/mock/gomock"
./pkg/storage/mocks/mocks.go:	gomock "github.com/golang/mock/gomock"
./pkg/client/introduce/mocks/mocks.go:	gomock "github.com/golang/mock/gomock"